R/date_input.R

```diff
@@ -43,9 +43,10 @@
 #'}
 #'
 #' @export
-date_input <- function(name, label = NULL, value = NULL, min = NULL, max = NULL,
+#' @rdname date_input
+date_input <- function(input_id, label = NULL, value = NULL, min = NULL, max = NULL,
```

Comment:
Is it possible to change the function, so it takes a character as an icon e.g. "dog" and do uiicon("dog") inside the function? We could then call shiny.semantic:::uiicon(), and don't export uiicon. Then the user would only have icon function exported.